problems/kattis/different/kotlin/sleddog/src/main/main.kt

```diff
@@ -1,14 +0,0 @@
-package sleddog
-
-fun main(args: Array<String>) {
-    var line = readLine();
-    while (line != null) {
-        var (a, b) = line.split(' ');
-        println(diff(a.toLong(), b.toLong()));
-        line = readLine();
-    }
-}
-
-fun diff(a: Long, b: Long): Long {
-    return Math.abs(a - b)
-}
```

Comment:
why were these files removed?

Reply:
@sleddog idk who those are I am only did the r2l.py